src/coffee/client.coffee

```diff
@@ -134,6 +134,7 @@ ALL_SERVICES = [
 #
 # Current options are available:
 # - `includeHeaders` will include some HTTP header information in the response, wrapped in a JSON object called `http`
+# - `maskSensitiveHeaderData` if set to false, it will filter out all http authorization headers from the response
```

Comment:
In my opinion this sounds better:

```If set to true, it will filter out all sensitive data like authorisation from response headers. By default it is set to false.````